alacritty/Cargo.toml

```diff
@@ -41,6 +41,7 @@ dirs = "2.0.2"
 
 [target.'cfg(not(any(target_os="windows", target_os="macos")))'.dependencies]
 x11-dl = "2"
+wayland-client = { version = "0.25", features = ["dlopen"] }
```

Comment:
Are there any problems with using multiple different `wayland-client` versions?

Just out of curiosity, do you know why winit isn't updated yet? I'd assume 0.25.0 is the minimum for frame callbacks?

I'd just like to make an effort to not duplicate dependencies unnecessarily.

Reply:
>Are there any problems with using multiple different wayland-client versions?

I don't want to use `wayland-client` winit is using, because I don't understand it at all and structure code with it is just hard even for such tasks. There're no problems, but one versions has good API, but the other one is just hard to use here.

>Just out of curiosity, do you know why winit isn't updated yet? I'd assume 0.25.0 is the minimum for frame callbacks?

Because it's hard to update to new wayland client for winit, however there's some ongoing work https://github.com/rust-windowing/winit/pull/1544 . Also winit is using SCTK, but we're using wayland-client here. Our clipboard will be updates to a new wayland-client soon as well though.

Reply:
>Since clipboard and winit are exchanging data types I think it's required for winit to also be bumped though.

This is not true, they exchange the `C` data type, which is directly related to `libwayland-client` and has nothing to do with `wayland-client`, so it's fine to mix them.